R/data.R

```diff
@@ -53,6 +53,13 @@
 #' @source \url{https://bitbucket.cdisc.org/projects/CED/repos/sdtm-adam-pilot-project/browse/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm/mh.xpt}
 "mh"
 
+#' Pharmacokinetics Concentrations Dataset
+#'
+#' A SDTM PC dataset simulated by Antonio Rodríguez Contestí
+#'
+#' @author Antonio Rodríguez Contestí
+"pc"
```

Comment:
add pp here and run devtools::document() to create man/pp.Rd

Reply:
thanks @toni-1991 for all the updates! just need to run devtools::document() to create man/pp.Rd now

Reply:
Hi @rossfarrugia,
I run the devtools::document() and added the created pp.Rd file

Reply:
Thanks @toni-1991